### DISCUSSION TITLE
I was able get line art quite accurate by modelling in such a way that it picks up the line art and adjusting line width node. How would I get closer results?

### DISCUSSION THREAD

**Discussion by rosstheblenderanimator** [bnpr/Malt#573] [Q&A] [answered]
I believe I can use vertex colours to at permanent lines not sure how. Don't want to grease pencil as it render behind a see through object and not as consistent.

![tumblr_ea4e25c012b12f97dc04e3e56e5d0535_4b6d3cc1_500](https://github.com/user-attachments/assets/b43c54d3-b1eb-4c1e-8a28-d307a0ba6488)

![woman base mesh attempt 14](https://github.com/user-attachments/assets/f46d8212-1750-4746-96f7-c9f49b4bb0ed)
![Uploading Greta_Expressions_Model.jpg…]()

**Comment by rosstheblenderanimator** [comment]
![Greta_Expressions_Model](https://github.com/user-attachments/assets/6c2d56f9-37f8-43ec-b246-2f72d809855f)

**Comment by rosstheblenderanimator** [comment]
This is my progress so far what do you think? 
![woman base mesh attempt 16](https://github.com/user-attachments/assets/5fdd12c0-323d-4193-ae13-66ab0c349917)

**Comment by rosstheblenderanimator** [comment, marked as answer]
This is the best I could do https://x.com/RossBergin/status/1825579004520251728

**Comment by pragma37** [comment]
If you want to have more precise control over where exactly your lines are drawn, this is probably the best method: #296

**Reply by rosstheblenderanimator** [reply]
Thanks I'm quite happy with result able to achieve with vertex colours using this node set up. Is there away to get line variation like PSOFT pencil +4?
![woman base mesh attempt 18 rigging ](https://github.com/user-attachments/assets/fd4bb43f-ec02-4fed-bc30-5f78d540bcc5)
